### DISCUSSION TITLE
Support request: Dashboard and CLI are broken after fresh install

### DISCUSSION THREAD

**Discussion by melgu** [aws/codewhisperer-command-line-discussions#111] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

I can install the Shell integration via the Dashboard.
The Dashboard doesn't throw lots of errors in the console.

### Actual behaviour

Unresponsive "Install" shell integration button.
Lots of errors in the console.
<img width="1072" alt="Bildschirmfoto 2024-03-19 um 03 22 16" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/9601403/41105587-cc69-4734-8880-dc8e01daa0de">

cw doctor and restart don't work.

### Steps to reproduce

Fresh install of CodeWhisperer.

### Environment

```yaml
fig worked without issues. CodeWhisperer doesn't work at all.
```

**Comment by grant0417** [comment]
Hey @melgu, sorry about the issue. This is fixed in 0.15.1 if you updated. Thanks for reporting!
